JSDemos/Demos/Lookup/EventHandling/description.md

```diff
@@ -1 +1,3 @@
-This demo illustrates how to&nbsp;perform a&nbsp;custom action when the lookup value is&nbsp;changed. Use the **onValueChanged** event handler to&nbsp;process the new lookup value.
\ No newline at end of file
+To handle the Lookup's value change, specify the [onValueChanged](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#onValueChanged) function. This demo uses it to display the picture and text for the selected item.
+
+The Lookup applies the selected value instantly. If you want users to click the Apply button instead, set the [applyValueMode](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#applyValueMode) property to *"useButtons"*. This functionality is not used in this demo.
```

Comment:
> If you want users to click the Apply button instead

This is not an end in itself. The thing is that the described option allows you to delay the effects of your change until a click on the "Apply" button.

> This functionality **is not used** in this demo.

This passive voice is totally avoidable from the look of it. For example:

> This demo does not show this functionality.